drivers/pcie/host/pcie.c

```diff
@@ -28,6 +30,10 @@ LOG_MODULE_REGISTER(pcie, LOG_LEVEL_ERR);
 #include <zephyr/drivers/pcie/controller.h>
 #endif
 
+#ifdef CONFIG_PCIE_PRT
+static bool prt_base_irq;
```

Comment:
You could add a code comment here explaining what this is for (i.e. that ACPI IRQ init was successful).